.eslintrc

```diff
@@ -24,7 +24,7 @@
         "ignoreParameters": true
       }
     ],
-    "@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true }],
+    "@typescript-eslint/no-unused-vars": ["warn", { "ignoreRestSiblings": true }],
```

Comment:
:eyes:

Reply:
@pouriaMaleki  This prevents having a work in progress branch. It didn't let us commit without removing all the work in progress variables, with a majority of 2 out of 3 devs we made it a warning :D I hope it's ok with you.

Reply:
Right :) you can also use `--no-verify` when committing to WIP Merge requests to avoid this.

Like: `git commit -m "message" --no-verify`

Reply:
If it's annoying, keep the "warn" and later we will perform this check in the pipelines when merging to the production branches.